src/ethereum-indexed-data/service-request/service-request.service.ts

```diff
@@ -180,7 +180,7 @@ export class ServiceRequestService {
         result.push(requestService._source);
       });
     } catch (error) { 
-      if (error.body.error.type === 'index_not_found_exception') {
+      if (error?.body?.error?.type === 'index_not_found_exception') {
         await this.logger.log(`API "service-requests/customer/{customerId}": ${error.body.error.reason}`)   
       } else {
         throw error        
```

Comment:
Next time, return directly after executing logger and refrain from using `else`.